src/components/ProgressBar/ProgressBar.test.tsx

```diff
@@ -0,0 +1,67 @@
+import { ProgressBar, ProgressBarProps } from "./ProgressBar";
+import { fireEvent } from "@testing-library/react";
+import { renderCUI } from "@/utils/test-utils";
+
+describe("Popover", () => {
```

Comment:
I suspect this was a copy paste. Describe `Progress bar`

Reply:
Sorry I copied the layout so that I only needed to write the tests